public/Partial/FAQ_PART/faq.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```

Comment:
Why is there is Partial folder in the public directory

Reply:
Bhaiya, that is a temporary folder, once all the components are ready, then that folder will be deleted.
Actually, it would be easier for us to collaborate.